src/app/components/big-input/big-input.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-big-input',
+  templateUrl: './big-input.component.html',
+  styleUrls: ['./big-input.component.scss']
+})
+export class BigInputComponent {
```

Comment:
I think we could create a module (`BigInputModule` ) for this component and export it.